src/chains-config/metadata-consts/polymeshConsts.ts

```diff
@@ -0,0 +1,8 @@
+import { MetadataConsts } from '../../types/chains-config';
+
+export const polymeshDefinitions: MetadataConsts[] = [
+	{
+		runtimeVersions: [2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022],
```

Comment:
we don't have 2020, 2021, 2022 ?

Reply:
Just future proofing. @adamdossa mentioned that the likelihood of extrinsic base weights changing with newer versions is very low.

If that does happen, this can be easily modified to address it